examples/3d/orthographic_zoom.rs

```diff
@@ -0,0 +1,106 @@
+//! Shows how to zoom and orbit an orthographic projection camera.
+
+use std::ops::Range;
+
+use bevy::{input::mouse::MouseWheel, prelude::*, render::camera::ScalingMode};
+
+// Multiply mouse movements by these factors
+const CAMERA_ORBIT_SPEED: f32 = 0.02;
```

Comment:
IMO these values are better stored as a resource in practice: that makes them easy to twiddle in various inspector tools.

Reply:
Old habits die hard :laughing: will do.